appinventor/appengine/war/WEB-INF/appengine-web.xml

```diff
@@ -115,7 +115,7 @@
     <!-- host[:port] to tell build server the app host url to callback
          when deploy AppInventor on other GAE implementation such as
          AppScale.Keep it blank when deploy on GAE -->
-    <property name="appengine.host" value="" />
+    <property name="appengine.host" value="localhost:8888" />
```

Comment:
I don't like this because we would have to reset it when deploying to production. I believe that the best option forward is to keep this empty.